src/status_im/ui2/screens/chat/home.cljs

```diff
@@ -232,9 +195,25 @@
         [list/flat-list
          {:key-fn    :first
           :data      (if (= @selected-requests-tab :received) received-requests sent-requests)
-          :render-fn received-contact-request/list-item}]]))])
+          :render-fn received-cr-item}]]))])
 
-(defn contact-requests [count]
+(defn get-display-name [{:keys [chat-id message]}]
+  (let [name        (first (<sub [:contacts/contact-two-names-by-identity chat-id]))
+        no-ens-name (str/blank? (get-in message [:content :ens-name]))]
+    (if no-ens-name
+      (first (str/split name " "))
+      name)))
+
+(defn requests-summary [requests]
```

Comment:
I believe this function should use translations with placeholders, like `hello {{name}}`, or at least not hardcode English words.